src/ui/pages/Overview/Positions/Positions.tsx

```diff
@@ -816,28 +800,22 @@ export function Positions({
 }) {
   const { currency } = useCurrency();
   const { ready, params, singleAddressNormalized } = useAddressParams();
-  const {
-    value: portfolioDecomposition,
-    isLoading: portfolioDecompositionIsLoading,
-  } = useAddressPortfolioDecomposition(
-    {
-      address: singleAddressNormalized,
-      currency,
-    },
-    { enabled: ready, client: useDefiSdkClient() }
+  const { data, ...portfolioQuery } = useWalletPortfolio(
+    { addresses: [params.address], currency },
```

Comment:
Is it ok that we changed `singleAddressNormalized` to `params.address`?

Reply:
Yeah, they have essentially been the same